script.js

```diff
@@ -9,13 +9,42 @@ const small_film_set = [
     { id: 6, title: "12 Angry Men", year: 1957, votes: 164558, rating: 8.9, rank: 6, category: "Western" }
 ]
 
+function saveItem() {
+    const validationResult = $$("form").validate();
+    if (validationResult) {
+        const item_data = $$("form").getValues();
+        $$("table").add(item_data);
+        $$("form").clear();
+        webix.message({
+            text: `Validation is successful`,
+            type: "success",
+            expire: 1000
+        });
+    }
+}
+
+function clearValidation() {
+    webix.confirm(`Do you realy want to clear validation?`)
+        .then(() => $$("form").clearValidation())
```

Comment:
it can be located in one then

.then(() => {
     $$("form").clearValidation();
     $$("form").clear();
})

Reply:
Thanks for example.